spec/tests/urbanopt_reopt_ghp_spec.rb

```diff
@@ -0,0 +1,126 @@
+# *********************************************************************************
+# URBANopt (tm), Copyright (c) Alliance for Sustainable Energy, LLC.
+# See also https://github.com/urbanopt/urbanopt-reopt-gem/blob/develop/LICENSE.md
+# *********************************************************************************
+require_relative '../spec_helper'
+require_relative '../../developer_nrel_key'
+require 'json-schema'
+
+
+RSpec.describe URBANopt::REopt do
+    run_dir = Pathname(__FILE__).dirname.parent / 'files' / 'run' / 'baseline_scenario_ghe'
```

Comment:
Using the [Pathname](https://ruby-doc.org/3.2.2/exts/pathname/Pathname.html) syntax simplifies, cleans, consolidates, and shortens a lot of file/path code. It works on Windows & Mac and is, imho, a big improvement over standard Ruby.